tests/frontend/test_search.py

```diff
@@ -282,6 +282,29 @@ def test_filter_extensions(base_url, selenium):
     assert len(search_page.result_list.themes) == 0
 
 
+@pytest.mark.nondestructive
+def test_filter_recommended_extensions_sort_by_users(base_url, selenium, variables):
+    page = Home(selenium, base_url).open()
+    search_page = page.search.search_for('')
+    sort = 'Most Users'
+    addon_type = 'Extension'
+    promoted = 'Recommended'
+    # apply filters and then search term
+    Select(search_page.filter_by_sort).select_by_visible_text(sort)
+    Select(search_page.filter_by_type).select_by_visible_text(addon_type)
+    Select(search_page.filter_by_badging).select_by_visible_text(promoted)
```

Comment:
What is your reasoning to have these strings are variables? You could insert the string into the ```select_by_visible_test``` function.

Reply:
Ok, I change the test format. :)